client/src/analysis/components/DefaultFrameView.tsx

```diff
@@ -15,68 +28,144 @@ const useDefaultFrameView = ({
     const availableModes = [
         {
             text: "Average",
-            value: AnalysisTypes.SUM_FRAMES,
+            value: DefaultModes.SUM,
+        },
+        {
+            text: "SD",
+            value: DefaultModes.SD,
         },
         {
-            text: "Pick",
-            value: AnalysisTypes.PICK_FRAME,
+            text: "PICK",
+            value: DefaultModes.PICK,
         },
         {
             text: "Average over ROI (disk)",
-            value: AnalysisTypes.SUM_FRAMES_ROI,
-        }
-    ];
+            value: DefaultModes.SUM_DISK,
+        },
+        {
+            text: "Average over ROI (rect)",
+            value: DefaultModes.SUM_RECT,
+        },
+        {
+            text: "SD over ROI (disk)",
+            value: DefaultModes.SD_DISK,
+        },
+        {
+            text: "SD over ROI (rect)",
+            value: DefaultModes.SD_RECT,
+        },
+    ]
 
-    const [frameMode, setMode] = useState(AnalysisTypes.SUM_FRAMES);
+    const [frameMode, setMode] = useState(DefaultModes.SUM);
 
     const frameModeSelector = <ModeSelector modes={availableModes} currentMode={frameMode} onModeChange={setMode} />
 
     const [cx, setCx] = React.useState(Math.round(scanWidth / 2));
     const [cy, setCy] = React.useState(Math.round(scanHeight / 2));
 
+
     const { coords: pickCoords, handles: pickHandles } = useFramePicker({
-        enabled: frameMode === AnalysisTypes.PICK_FRAME,
+        enabled: frameMode === DefaultModes.PICK,
         scanWidth, scanHeight,
         jobIndex: 0,
         analysisId,
         cx, cy, setCx, setCy
     });
 
-    const { sumRoiHandles, sumRoiWidgets } = useRoiPicker({
-        enabled: frameMode === AnalysisTypes.SUM_FRAMES_ROI,
+
+    const { RectRoiHandles, RectRoiWidgets, RectroiParameters }  = useRectROI({ scanHeight, scanWidth })
+    const { diskRoiHandles, diskRoiWidgets, diskroiParameters}  = useDiskROI({ scanHeight, scanWidth })
+
+    useRoiPicker({
```

Comment:
Hmm. There are now so many different `useRoiPicker` calls in here, when in reality, we basically only have a) two different analyses (sum, sd) and three or four types of ROI (whole dataset, disk, rect, "point" in the case of pick). Maybe we can work more on decoupling ROI selection and running the analysis?